Tests/test_file_jpeg.py

```diff
@@ -898,6 +898,22 @@ def test_getxmp(self):
             with Image.open("Tests/images/hopper.jpg") as im:
                 assert im.getxmp() == {}
 
+    def test_getxmp_no_prefix(self):
+        with Image.open("Tests/images/xmp_no_prefix.jpg") as im:
+            if ElementTree is None:
+                with pytest.warns(UserWarning):
```

Comment:
Why is there a `UserWarning`?

Reply:
`ElementTree` is None if defusedxml is unavailable, triggering this warning.
https://github.com/python-pillow/Pillow/blob/1e7dcd3aeb3b10b802085b1ccc37ccc619f8bcfe/src/PIL/Image.py#L1410-L1411

Reply:
Sure, done, and I've added it to other locations as well.